bin/newver

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+set -eu
+
+if [ $# -eq 0 ]; then
+  >&2 echo "Usage: 'bin/newver X.Y.Z', where X.Y.Z is the Rubocop version to add."
+  exit 1
+fi
+NEWVER="$1"
+
+git checkout Gemfile*
+git checkout -b "newver/${NEWVER}"
+
+ed Gemfile <<-EDITS
+/gem "parser",/c
+gem "parser"
+.
+/gem "rubocop",/c
+gem "rubocop", "~> ${NEWVER}", require: false
+.
+wq
+EDITS
+
+bundle update
```

Comment:
should we do something more scoped like, `bundle update rubocop`?

Reply:
Note: in order for this to work, it would be necessary to add a `make image` before the `ed` line

Reply:
Whoops, this comment was meant to go on the thread about using docker, not the thread about scoping the bundle update :sweat_smile: